---
title: "Bioinformatics Core Facility"
subtitle: 'Website Test'
output:
  bookdown::html_document2:
    toc: true
    toc_float:
      collapsed: true
      smooth_scroll: true
      print: false
    toc_depth: 4
    number_sections: true
    highlight: tango
    df_print: paged
    code_folding: "show"
    self_contained: true
    keep_md: false
    encoding: 'UTF-8'
    css: "assets/report.css"
---

```{r,child="assets/_header-lab.Rmd"}
```

<!-- ----------------------- Do not edit above this ----------------------- -->

# Bioinformatics


The BRIC Bioinformatics Core Facility supports researchers at the BRIC center, the University of Copenhagen (UCPH) and external institutions. 


The Facility is composed of a multidisciplinary team updated on the latest technologies in molecular biology . 


**Multiple services are offered:** \
1. Bioinformatic analysis services (with a priority to BRIC users), \
2. Access to a maintained high-performance computational server (only to BRIC users) and\
3. Training in bioinformatics (only to BRIC users).


```{r fig-core, echo = F, eval = T }
knitr::include_graphics('/Users/flb202/Documents/GitHub/BRIC_Web/Core/Core3.png')
```



# Our Services

Our support involves **consultation**, **experimental desing**, **data analysis**, **guidance**, **short & long-term projects** and **training**.

## **Consultation**

You are always welcome to [talk to us]! We can work together to understand your needs.


<span style="color:black"> **No fee is applied to understand your needs**</span>

## **Experimental Desing**

Including a bioinformatician in an early stage of your project can save you time, work, and money.

## **Data Analysis**


We adapt to your needs. We can discuss and find together the best way to analyze your data. Alternatively, if you want to analyze the data yourself we can also provide support for your bioinformatic analysis with your own code or with existing pipelines.


## **Standard bioinformatics project**

**A plus in using our services:** We work with optimized pipelines ensuring reproducibility and providing reports.


<span style="color:black"> **Estimating hours is always challenging; We estimate a workload of 20 to 30 hrs for a standard analysis of simple datasets of the following types:**</span>

1. Gene Expression \
1.1 RNA-seq \
1.2 ChIP-seq \

2. ATAC-seq

3. Single Cell

**An Example of a Standard RNA-seq dataset analysis**

>. Initial meeting / communications
>
>. Reading about the project
>
>. Transfer raw sequencing data
>
>. Quality assessment of the data 
>
>. Communication with the user
>
>. Alignment of sequencing reads onto the genome of interest and gene quantification
>
>. Differential gene expression analysis
>
>. Functional enrichment
>
. Report and Data visualization
>
. Meetings to discuss results and possible follow-ups

## **End-To-End**

We aim to be able to offer an End-to-End service for your project. We work in close collaboration with other BRIC core facilities to offer and End-To-End service \
**Library Preparation - Sequencing - Bioinformatics** \

 Please, [Talk to us] for more information on how to get the best of us.

## More services

- You can also request our services for **Data Management**. We can prepare and submit your data to any public database.

- Don't hesitate to contact us to help you preparing your **manuscript** or **grant application**.

# Prices
BRIC users: 400 DKK/hour

Other KU users: 500 DKK/hour

External users: 980 DKK/hour

# About us

```{r, include=FALSE}
library(png)
library(grid)
library(gridExtra)
img1 <-  rasterGrob(as.raster(readPNG("/Users/flb202/Documents/KU/BRIC/Core/Leonor.png")), interpolate = FALSE)
img2 <-  rasterGrob(as.raster(readPNG("/Users/flb202/Documents/KU/BRIC/Core/Ju.png")), interpolate = FALSE)
grid.arrange(img1, img2, ncol = 2)
```


```{r fig-l, echo = F, eval = T }
knitr::include_graphics('/Users/flb202/Documents/GitHub/BRIC_Web/Core/Leonor.jpg')
```

**Leonor Rib, PhD ** \
leonor.rib@bric.ku.dk \

I'm a Computer Scientist with a passion for biology. I hold a PhD in computational biology after doing a PhD in Integrated experimental and computational biology in Switzerland. I have extensive experience in the experimental design and analysis of multiomics and genetic data for more than 15 years in collaboration with hospitals, academic research groups and biotech companies. More details can be found in my [LinkedIn profile](https://www.linkedin.com/in/leonor-rib/) and [Google scholar](https://scholar.google.com/citations?user=iSRLtQcAAAAJ&hl=en).

*I'm originally from Spain, in Catalonia. And if you offer me a ["Calçotada"](https://theculturetrip.com/europe/spain/articles/calcotada-a-guide-to-the-catalan-onion-craze/) in good company and a good wine with ["Porró"](https://www.businessinsider.com/how-people-drink-wine-in-barcelona-2016-5?r=US&IR=T) you will make me happy * :)

```{r fig-eu, echo = F, eval = T }
knitr::include_graphics('/Users/flb202/Documents/GitHub/BRIC_Web/Core/Ju.png')
```

**Juliana Assis, PhD**

juliana.geraldo@bric.ku.dk \

I'm a Biologist with a Ph.D. in Bioinformatics. My field lies broadly in multi-omics data such as Genomic, Transcriptomic, Metagenomic, Metabolomics, and Data Integration.
I have extensive experience developing the strategy/experimental design and scientific skills for NGS analysis and long-reads. Making Reproducibility in research is my main focus of study.

You can find more about my work here 
[Google Scholar](https://scholar.google.com.br/citations?user=MYu19awAAAAJ&hl=en), and, here: [GitHub](https://github.com/Juassis)
 

*My home country is Brazil, and I'm very proud of my cooking skills and my Caipirinha!*


**Linbu Liao, Ph.D.**

```{r fig-lin, echo = F, eval = T }
knitr::include_graphics('/Users/flb202/Documents/GitHub/BRIC_Web/Core/Linbu.png')
```

linbu.liao@bric.ku.dk \

I hold a Ph.D. in Bioinformatics and am a Bioinformatician. As a member of the CellX project, I am tasked with developing Spatial Transcriptomics tools to better comprehend how cells communicate with one another. My relevant work is [SC2Spa, a deep learning-based tool to map transcriptome to spatial origins at cellular resolution](https://sc2spa.readthedocs.io/en/latest/).

*I'm Chinese. In China, I received a bachelor's degree in petroleum engineering as well as a master's degree in software engineering. I made substantial adjustments in both disciplines and locations. Staying on the righteous path is more important to me than being who I am supposed to be.*



# Talk to us

BRIC - Biotech Research & Innovation Centre \
Ole Maaløes vej 5 \
DK-2200 Copenhagen \
Office: Biocenter 1-3-20 \

E-mail: bioinf@bric.ku.dk


<!-- --------------------- Do not edit this and below ---------------------- -->

</br>

```{r,child="assets/_footer-lab.Rmd"}
```

```{r,eval=FALSE,echo=FALSE}
# manually run this to render this document to HTML
rmarkdown::render("website.Rmd")
# then run this to convert HTML to PDF (if needed)
#pagedown::chrome_print("website.html",output="website.pdf")
```
